pkgs/ninja-build/ninja/registry.yaml

```diff
@@ -11,4 +11,20 @@ packages:
     supported_envs:
       - darwin
       - amd64
+    version_constraint: semver("< 1.12.0")
+    version_overrides:
+      - version_constraint: semver(">= 1.12.0")
```

Comment:
Please follow the style guide.

https://aquaproj.github.io/docs/develop-registry/registry-style-guide#version_overrides-style-guide